extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -428,7 +443,7 @@ public <T> Page<T> listEntities(
       List<T> resultsOrEmpty =
           results == null
               ? Collections.emptyList()
-              : results.stream().filter(entityFilter).map(transformer).collect(Collectors.toList());
+              : results.stream().map(transformer).collect(Collectors.toList());
       return Page.fromItems(resultsOrEmpty);
```

Comment:
It's not part of this PR, but `Page.fromItems()` always uses `DonePageToken`. This looks very confusing... I'll try and review closer again later :thinking:

Reply:
Would it make sense to rename `Page.fromItems()` to `Page.finalPage()`?